unified_planning/io/ma_pddl_writer.py

```diff
@@ -101,13 +119,15 @@ def __init__(
         self,
         problem: "up.model.multi_agent.MultiAgentProblem",
         explicit_false_initial_states: Optional[bool] = False,
+        unfactored: Optional[bool] = False,
```

Comment:
Can you document this new parameter explaining the main differences in the generated files?